ringbuffer.py

```diff
@@ -253,7 +263,7 @@ def force_reader_sync(self):
                 reader.set(writer_position.counter)
 
             for reader in self.readers:
-                p = reader.get()
+                reader.get()
```

Comment:
Is this loop even necessary? We can wait to synchronize `reader.position` later, right?

(also, do you know why we have the `counter` is a `ctypes.c_longlong` thing? Is it a performance optimization, or something?)

Reply:
Not sure about the loop. I think the counter needs to work across process boundaries, thus a rawvalue is required.